src/plugins/home/public/application/components/tutorial_directory.js

```diff
@@ -178,6 +179,7 @@ class TutorialDirectoryUi extends React.Component {
   renderTabs = () => {
     return this.tabs.map((tab, index) => (
       <EuiTab
+        data-test-subj={tab.id}
```

Comment:
To avoid risk of conflicts with potential other `data-test-subj` on the page, I would prefix that, like `homeTab-${tab.id}`.

Same for the tutorials a few lines down.

Reply:
thanks @pgayvallet done.